qa/tasks/cephfs/filesystem.py

```diff
@@ -569,7 +572,7 @@ def get_pool_df(self, pool_name):
     def get_usage(self):
         return self._df()['stats']['total_used_bytes']
 
-    def are_daemons_healthy(self):
+    def are_daemons_healthy(self, fs_name=None):
```

Comment:
How about move this method to MDSCluster. Then add a wrapper method in Filesystem which as a default fscid.

I also think it's best to avoid using fs names in these methods where possible. Instead prefer fscid.